illumos-utils/src/zfs.rs

```diff
@@ -8,8 +8,8 @@ use crate::{execute, PFEXEC};
 use std::fmt;
 use std::path::PathBuf;
 
-pub const ZONE_ZFS_DATASET_MOUNTPOINT: &str = "/zone";
-pub const ZONE_ZFS_DATASET: &str = "rpool/zone";
+pub const ZONE_ZFS_RAMDISK_DATASET_MOUNTPOINT: &str = "/zone";
+pub const ZONE_ZFS_RAMDISK_DATASET: &str = "rpool/zone";
```

Comment:
I went ahead and renamed these constants to make it a little more clear that "anything you put here is on a ramdisk, and will not last across reboots!"